packages/nodes-base/nodes/Wise/GenericFunctions.ts

```diff
@@ -65,6 +70,95 @@ export async function wiseApiRequest(
 	}
 }
 
+/**
+ * Make an authenticated SCA API request to Wise.
+ */
+ export async function wiseSCAApiRequest(
+	this: IHookFunctions | IExecuteFunctions | ILoadOptionsFunctions,
+	method: 'GET' | 'POST' | 'PUT' | 'DELETE' | 'HEAD',
+	endpoint: string,
+	body: IDataObject = {},
+	qs: IDataObject = {},
+	option: IDataObject = {},
+) {
+	const { apiToken, environment, privateKey } = await this.getCredentials('wiseApi') as {
+		apiToken: string,
+		environment: 'live' | 'test',
+		privateKey: string | null,
+	};
+
+	const rootUrl = environment === 'live'
+		? 'https://api.transferwise.com/'
+		: 'https://api.sandbox.transferwise.tech/';
+
+	const options: IHttpRequestOptions = {
+		headers: {
+			'user-agent': 'n8n',
+			'Authorization': `Bearer ${apiToken}`,
+		},
+		method,
+		url: `${rootUrl}${endpoint}`,
+		qs,
+		body,
+		json: true,
+		returnFullResponse: true,
+		validateStatus: (status) => {
+			return (status >= 200 && status < 300) || status === 403; // default
+		},
+	};
+
+	if (!Object.keys(body).length) {
+		delete options.body;
+	}
+
+	if (!Object.keys(qs).length) {
+		delete options.qs;
+	}
+
+	if (Object.keys(option)) {
+		Object.assign(options, option);
+	}
+
+	let response;
+	try {
+		response = await this.helpers.httpRequest!(options);
+	} catch (error) {
+		throw new NodeApiError(this.getNode(), error);
+	}
+
+	if (response.statusCode === 200) {
+		return response.body;
+	}
+
+	if (response.statusCode === 403 && response.headers['x-2fa-approval']) {
+		if (!privateKey) {
+			throw new NodeApiError(this.getNode(), {...response.body, message: 'This request requires Strong Customer Authentication (SCA). Please add a key pair to your account and n8n credentials. See https://api-docs.transferwise.com/#strong-customer-authentication-personal-token'},);
+		}
+		// Sign the x-2fa-approval
+		const oneTimeToken = response.headers['x-2fa-approval'] as string;
+		const signerObject = createSign('RSA-SHA256');
+		signerObject.update(oneTimeToken);
+		const signature = signerObject.sign(
+			privateKey,
+			'base64',
+		);
+		delete options.validateStatus;
+		options.headers = {
+			...options.headers,
+			'X-Signature': signature,
+			'x-2fa-approval': oneTimeToken,
+		};
+		try {
+			response = await this.helpers.httpRequest!(options);
+			return response.body;
+		} catch (error) {
+			throw new NodeApiError(this.getNode(), error);
+		}
+	} else {
+		throw new NodeApiError(this.getNode(), {...response.body},);
```

Comment:
Why make a copy of the body? And why only the body? If this else branch is reached, it looks like the only possibility is that it was a 403 lacking the expected SCA header, so it'd be useful for the error to contain the received headers.